PythonDZ/dz2.py

```diff
@@ -0,0 +1,22 @@
+# Напишите программу, которая получает целое число и возвращает его шестнадцатеричное строковое представление.
+# Функцию hex используйте для проверки своего результата.
+
+from copy import copy
+
+
+num = int(input('Задача 2. Введите число:\n'))
+
+hexadecimal_sys = 16
+res = []
+numbers = '0123456789abcdef'
+while abs(num) > 0:
```

Comment:
А зачем вам функция `abs`?
```suggestion
while abs:
```